util/shared/tokenHelper.js

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
The token logic is definitely not just a "util" method. It should be in authentication module. Util is for just general method, like string, math, wrapper, pure convenient method. It should not contain any business logic. 
Especially secret is very important and specific here.